src/libs/DateUtils.js

```diff
@@ -110,6 +110,14 @@ function startCurrentDateUpdater() {
     });
 }
 
+/*
+ * get the time using user's timezone for chat start screen.
+ */
+export const currentTimeStampFromProvidedZone = (userTimeZone, format) => {
```

Comment:
Don't use export and const for functions. See the Style guide or follow older code.